openmdao/core/explicitcomponent.py

```diff
@@ -87,6 +88,18 @@ def _setup_partials(self, recurse=True):
             self._declare_partials(out_name, out_name, rows=arange, cols=arange,
                                    val=np.full(meta['size'], -1.))
 
+    def _setup_jacobians(self, recurse=True):
```

Comment:
why did this get added to ExplicitComp?

Reply:
This is just a minor optimization basically.  Since explicit comps don't have linear solvers, the entire setup_jacobians function goes away except for the call to _set_partials_meta.